Dockerfile

```diff
@@ -26,14 +26,21 @@ RUN apt-get -y update \
 # Required by tonic
 RUN rustup component add rustfmt
 
+# Build UI
+COPY quickwit-ui /quickwit/quickwit-ui
+
+WORKDIR /quickwit/quickwit-ui
+
+RUN echo "Building Quickwit UI" \
+    && npm install --location=global yarn \
+    && yarn install \
+    && yarn build
```

Comment:
One slightly lame thing here, is that the logic used to build is now duplicated in the makefile and in the Dockerfile. It might yield to long-to-debug problems if we modify one and not the other.

Reply:
Yes, agreed.